src/bin/stratisd.rs

```diff
@@ -157,6 +157,18 @@ fn trylock_pid_file() -> StratisResult<File> {
     }
 }
 
+/// Return desired timeout based on if dbus timeout should be selected, non-eventable timeout
+/// selected or default selected
```

Comment:
I believe that this function should be better documented, to explain the meaning of the arguments and also the reason behind the choices of the return values. This will help understand if it is called correctly and also inform any subequent revisions of the values that may become necessary.

Reply:
With Andy's comment at the meeting, I think we can make this much simpler.